packages/dapp/src/components/Dashboard/DashboardActivity.tsx

```diff
@@ -339,6 +364,14 @@ class DashboardActivity extends React.Component<
     );
   };
 
+  private renderTransferBalance = (value: number) => {
+    if (value === 0 && this.state.transferTokenToVoting === false) {
```

Comment:
Maybe we should use the `BalanceType` enum values in a state property (maybe call it `fromBalanceType` or `sourceBalanceType`?) here instead of this `transferTokenToVoting` boolean.  

Then this whole function could just be:

```
private renderTransferBalance = (value: number) => {
  this.setState({ fromBalanceType: value });
};
```

And the above ternary could be:

```
{this.state.fromBalanceType === BalanceType. AVAILABLE_BALANCE ? <DepositTokens /> : <ReclaimTokens />}
```